lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -85,6 +85,76 @@ class StandardAccountTest {
         StandardAccount(Felt.ZERO, privateKey, provider, chainId)
     }
 
+    @Test
+    fun `creating cairo 0 account with automatic version determination`() {
```

Comment:
nit:
```suggestion
    fun `cairo 0 account with automatic version detection`() {
```
The same suggestion below